src/plugins/usage_collection/common/types/stats/core_metrics.ts

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * The types are exact duplicates of the ops metrics types declared in core needed for reporting in the stats api.
```

Comment:
We're duplicating the [types from `core`](https://github.com/elastic/kibana/blob/f567ab6f3ce4521b301d17dec9ef4e4a2c24d8b1/packages/core/metrics/core-metrics-server/index.ts#L9-L19) so that we'll detect any changes to the original metrics.